### PR TITLE
[imp] web: in mobile view it should open numeric keyboard in number f…

### PR DESCRIPTION
**Pull request by vas-odoo**
TASK :- https://www.odoo.com/web#id=35231&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
PAD :- https://pad.odoo.com/p/r.d1777ce29d89e8c82d6d995e46fcdc09

…ields

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/xml/base.xml [hunk 1]
```diff
@@ -1028,7 +1028,7 @@
 <div t-name="SearchView" class="o_searchview">
     <span class="o_searchview_more fa" title="Advanced Search..."/>
 </div>
-<input t-name="SearchView.InputView" type="text" class="o_searchview_input" placeholder="Search..."/>
+<input t-name="SearchView.InputView" type="search" class="o_searchview_input" placeholder="Search..."/>
```
Comment (xmo-odoo) [inline addons/web/static/src/xml/base.xml]: No. type=search leads to browser-specific styling changes which we do not want.

File: addons/web/static/src/js/fields/basic_fields.js [hunk 2]
```diff
@@ -718,6 +726,14 @@ var FieldInteger = InputField.extend({
     isSet: function () {
         return this.value === 0 || this._super.apply(this, arguments);
     },
+    _renderEdit: function () {
+        this._super.apply(this, arguments);
+        if( this.$input ){
```
Comment (xmo-odoo) [inline addons/web/static/src/js/fields/basic_fields.js]: Why? Is there *any* situation where InputField would not have created the $input and we'd still expect the field to work correctly?

File: addons/web/static/src/js/fields/basic_fields.js [hunk 3]
```diff
@@ -718,6 +726,14 @@ var FieldInteger = InputField.extend({
...
+            this.$input.attr({
```
Comment (xmo-odoo) [inline addons/web/static/src/js/fields/basic_fields.js]: Use the simpler two-parameters form (`.attr(key, value)`).

**Comment by aab-odoo** [comment]
Hello,

Could you add test cases?

Thanks.

**Comment by rdeodoo** [comment]
The task was cancelled and there is conflicts now
Reopen if this is still a thing :)

**Pull request closed without merging**